src/WebdriverClassicDriver.php

```diff
@@ -91,18 +92,26 @@ class WebdriverClassicDriver extends CoreDriver
 
     private ?string $initialWindowHandle = null;
 
+    /**
+     * @var TDriverInstantiator
+     */
+    private $driverInstantiator;
+
     /**
      * @param string $browserName One of 'edge', 'firefox', 'chrome' or any one of {@see WebDriverBrowserType} constants.
      * @param TCapabilities $desiredCapabilities
+     * @param TDriverInstantiator $driverInstantiator
      */
     public function __construct(
         string $browserName = self::DEFAULT_BROWSER,
         array $desiredCapabilities = [],
-        string $webDriverHost = 'http://localhost:4444/wd/hub'
+        string $webDriverHost = 'http://localhost:4444/wd/hub',
+        ?callable $driverInstantiator = null
```

Comment:
We're in a driver class and the parameter is called `$driverInstantiator`. Shouldn't it be called `$webDriverInstantiator`?